src/dotnet-interactive/CommandLine/CommandLineParser.cs

```diff
@@ -210,32 +215,29 @@ Command Jupyter()
                     }.ExistingOnly()
                 };
 
-                command.Handler = CommandHandler.Create<StartupOptions, JupyterOptions, IConsole, InvocationContext, CancellationToken>(JupyterHandler);
+                jupyterCommand.Handler = CommandHandler.Create<StartupOptions, JupyterOptions, IConsole, InvocationContext, CancellationToken>(JupyterHandler);
 
-                var installCommand = new Command("install", "Install the .NET kernel for Jupyter")
+                var installSubCommand = new Command("install", "Install the .NET kernel for Jupyter")
                 {
                     httpPortRangeOption,
                     pathOption
                 };
 
-                installCommand.Handler = CommandHandler.Create<IConsole, InvocationContext, HttpPortRange, DirectoryInfo>(InstallHandler);
+                installSubCommand.Handler = CommandHandler.Create<IConsole, InvocationContext, HttpPortRange, DirectoryInfo>(InstallHandler);
```

Comment:
This should probably be called `installCommand` since these are all subcommands.